mortgageAPI.yaml

```diff
@@ -2582,6 +2582,8 @@ components:
                 - divorced
                 - widowed
                 - registered-partnership
+                - legally separated
+                - Partnership dissolved
```

Comment:
This does not follow the general style of enums in the API (no spaces, no capitals). I'd suggest, though:

```
- legally-separated
- partnership-dissolved
```

Reply:
you are right. done